collections/_contributors/saurabh-thakur.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: person
+name: Saurabh Thakur
+position: Contributor
+image_path: /images/photos/Saurabh Thakur.jpg
+twitter: thakursaurabh98
+github: thakursaurabh1998
+blurb: Mesh Head
```

Comment:
Blurb can simply be `Contributor`

Reply:
That is default set to Mesh Head for every contributor.
What you are talking about is defined in position..

Reply:
Ohh yeah sorry!